bundle-size-chart/static/index.html

```diff
@@ -30,7 +30,7 @@
 <body>
   <h1><a href="https://github.com/ampproject/amphtml">AMPHTML</a> bundle-size history</h1>
   <div id="chart"></div>
-  <label for="filter">Filter</label> <input type="text" id="filter" value="v0.js">
+  <label for="filter">Filter (for example, "v0.js")</label> <input type="text" id="filter" value="">
```

Comment:
This is gonna make the chart unreadable in the default view...

Reply:
I don't entirely agree. Right now, most information in the chart is hidden.

With this change: For interactive sessions (opening it in a browser), hovering over the chart will highlight different lines, and for non-interactive sessions (showy), you will still see more useful information, particularly inflections in various lines.

WDYT?

![bhL3UqsWDLH](https://user-images.githubusercontent.com/26553114/70168079-1b785d80-1696-11ea-91aa-a1bf7cd21d11.png)

Reply:
Bump.

Reply:
@danielrozenberg ^

Reply:
Maybe it makes sense to link this with https://github.com/ampproject/amphtml/blob/master/build-system/tasks/bundle-size/APPROVERS.json and only display those files that are there, to encourage WGs to add their files in there ;)

Reply:
I like your idea much more than what I was trying to do in this PR. Will abandon this.